bucket/jaspersoft-studio.json

```diff
@@ -0,0 +1,41 @@
+{
+    "homepage": "https://community.jaspersoft.com/project/jaspersoft-studio",
+    "description": "The Eclipse-based report development tool for JasperReports and JasperReports Server",
+    "license": "EPL-1.0",
+    "version": "6.6.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://downloads.sourceforge.net/project/jasperstudio/JaspersoftStudio-6.6.0/TIB_js-studiocomm_6.6.0_windows_x86_64.zip",
+            "hash": "sha1:aef64a51aa7ab2d4d7aa91d8269d070f24d40637"
+        },
+        "32bit": {
+            "url": "https://downloads.sourceforge.net/project/jasperstudio/JaspersoftStudio-6.6.0/TIB_js-studiocomm_6.6.0_windows_x86.zip",
+            "hash": "sha1:61e87c4e1b619e0920af13eef70a90de9e486132"
+        }
+    },
+    "extract_dir": "jaspersoftstudio",
+    "shortcuts": [
+        [
+            "Jaspersoft Studio.exe",
+            "Jaspersoft Studio"
+        ]
+    ],
+    "persist": [
+        "configuration",
+        "Jaspersoft Studio.ini"
+    ],
+    "checkver": {
+        "url": "https://community.jaspersoft.com/project/jaspersoft-studio/releases",
+        "re": "Jaspersoft Studio ([\\d.]+)"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://downloads.sourceforge.net/project/jasperstudio/JaspersoftStudio-6.6.0/TIB_js-studiocomm_$version_windows_x86_64.zip"
```

Comment:
Update URLs fixed to version `6.6.0`.
Fixed in https://github.com/lukesampson/scoop-extras/commit/79c9a396844e4cb613eed8c5ce6ada441bcf9414#diff-166aa98f8660ba39b87828fe963f63e5.